### ISSUE TITLE
Design link to dev environment

### ISSUE THREAD

**Issue by fantandrea** [Linkurious/documentation#100]
In the style section of production doc link (both [latest](https://doc.linkurio.us/admin-manual/latest/styles/) and [2.8.1](https://doc.linkurio.us/admin-manual/2.8.1/styles/)) the Design link points to the dev env.

![image](https://user-images.githubusercontent.com/12516532/64941411-eadb1180-d866-11e9-9677-63c20714ddf7.png)

**Comment by fantandrea** [comment]
@davidrapin the issue is actually related to hardcode links in the `content\admin\dokapi.json` configuration file:

```json
{
  ...,
  "variables": {
    ...,
    "apidoc.url": "https://doc.linkurio.us/server-sdk/latest/apidoc/",
    "design.url": "https://dev.linkurio.us/doc/user-manual/latest/design/",
    ...,
  }
}
```

What is the logic behind porting this in prod? Should I just reference the prod environment also in the second link? (both links in test environment will always target to the online site)

**Comment by davidrapin** [comment]
- we should avoid hardcoded links in the code (even using variables) as
much as possible
- hardcoded links should never be to the dev env (it is not public)

next step:
- fix the link
- find the person who did this and explain why it should not be done

**Comment by francescoinfante** [comment]
Unsure how it happened but it's actually me 11 months ago :p 

https://github.com/Linkurious/documentation/commit/1aa864d07eef6fafb50f583f0c04b670a4bc925f

going to fix that

**Comment by francescoinfante** [comment]
@davidrapin the reason we need to use hardlink is the following:
https://github.com/Linkurious/documentation/issues/58

**Comment by francescoinfante** [comment]
fixed for 2.8.2
